src/signalingclient/SignalingClientJoin.ts

```diff
@@ -9,12 +9,11 @@ import ApplicationMetadata from '../applicationmetadata/ApplicationMetadata';
 export default class SignalingClientJoin {
   /** Initializes a SignalingClientJoin with the given properties.
    *
-   * @param maxVideos The maximum number of video tiles to send.
    * @param sendBitrates Whether the server should send Bitrates messages.
    * @param applicationMetadata [[ApplicationMetadata]].
    */
+
```

Comment:
a bit of a stretch but since this class is public, we cannot really delete this param.

Reply:
ohh interesting. I can add a `@deprecated` parameter to it and log an error in-case anyone tries to use that parameter.

Reply:
Could we document how it works before and now what would be and why we deprecate it?

Reply:
Updated the comments